HLTrigger/Configuration/python/Phase2/ESProducersSources/ttrhbwr_cfi.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+
+ttrhbwr = cms.ESProducer("TkTransientTrackingRecHitBuilderESProducer",
```

Comment:
instances of ```TkTransientTrackingRecHitBuilderESProducer``` come with an hlt prefix in confdb (looking at HLT_FULL)

Reply:
And also here now.